### ISSUE TITLE
CLI version?

### ISSUE THREAD

**Issue by Sandvich** [iBroadcastMediaServices/MediaSyncLiteLinux#3]
Hi,

I've started using ibroadcast as a replacement for google music, and I want a backup of my library stored on my server. This means that it's a lot easier to keep things entirely on the server, and not on my own laptop.

As it is, there's no way for me to do this as mediasynclite requires a GUI. Are there plans to include a purely CLI mode?

**Comment by flagoworld** [comment]
Hi @Sandvich,

We have a a number of uploads scripts in different languages. The python script should let you upload those files. It can be found here:

https://project.ibroadcast.com/

**Comment by Sandvich** [comment]
Thanks for the reply. I tried out the python version as you suggested, and I found a bug. Is it also in a repo that I can PR?

In case it isn't, I'll explain the bug here.

Line 105 of the script, in `load_files(self, directory=None)`:
```python
self.files = []
```
Because the function is recursive, this gets reset every time the function enters a new directory. I moved it to the end of `login` (line 89) and it functions as expected.
